.gitignore

```diff
@@ -1,5 +1,6 @@
 node_modules
 lib
+dist
```

Comment:
files inside `dist` directory are being using for CDN purposes. Look here: [jsdelivr](https://cdn.jsdelivr.net/npm/maticjs/) and [unpkg](https://unpkg.com/maticjs/dist/)

Reply:
Also, check [CDN usage](https://github.com/maticnetwork/matic.js#cdn) in README